sirepo_bluesky/sirepo_flyer.py

```diff
@@ -368,9 +377,10 @@ def collect(self):
                    'filled': {key: False for key in data}}
 
     @staticmethod
-    def _run(sim, return_status):
+    def _run(sim, return_status, return_duration, start_time):
```

Comment:
The `start_time` arg does not seem to be used anywhere. I think it can be removed, and should not be passed in the above methods calling `_run(...)`.